tools/type_whisperer/typedb_gen.py

```diff
@@ -20,8 +20,9 @@
     ('envoy\.type\.matcher', 'envoy.type.matcher.v3'),
     ('envoy\.type', 'envoy.type.v3'),
     ('envoy\.config\.cluster\.redis', 'envoy.extensions.clusters.redis.v3'),
-    ('envoy\.config\.retry\.previous_priorities',
-     'envoy.extensions.retry.priority.previous_priorities.v3'),
+    (
+        'envoy\.config\.retry\.previous_priorities',
+        'envoy.extensions.retry.priority.previous_priorities.v3'),
```

Comment:
this is kinda annoying

Reply:
after playing with the "knobs" i cant seem to find any that prevent this (other than disabling `split_before_first_argument` altogether)

i have raised a ticket here https://github.com/google/yapf/issues/907